app/scripts/modules/appengine/serverGroup/configure/wizard/basicSettings.html

```diff
@@ -59,63 +59,128 @@
       </div>
     </div>
 
-    <div class="form-group">
-      <div class="col-md-3 sm-label-right">
-        Git Repository URL <help-field class="help-field-absolute" key="appengine.serverGroup.repositoryUrl"></help-field>
+    <div class="form-group row">
+      <label class="col-md-3 sm-label-right">Source Repository Type</label>
+      <div class="col-md-9">
+        <select class="input-sm"
```

Comment:
For a short fixed list of options, it's better to use radio buttons instead of a select. It's one click instead of two to select an option, and you can see all the options immediately.
See https://github.com/spinnaker/deck/blob/master/app/scripts/modules/appengine/loadBalancer/configure/wizard/stageAllocationConfigurationRow.component.html#L35-L50

Reply:
Also, you should be careful about remains on the command when you change `sourceType`.

For example, if I am using a `git` type and have a `sourceRepository` of `https://myGitRepo.git`, that value is going to show up inside the `GCS URL` input field if I change `sourceType` to `gcs`. The easiest thing is to just clear `sourceRepository` when you change `sourceType`.